docs/content/docs/recipes/teams.md

```diff
@@ -3,358 +3,5 @@ title: Using Fluid with Microsoft Teams
 menuPosition: 4
 ---
 
-This is a tutorial for integrating Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs). By the end of this tutorial you will be able to integrate any Fluid-powered applications into Microsoft Teams and collaborate with others in real-time.
+Refer to the [Use Fluid with Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-fluid-msteam) documentation for steps to integrate Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/what-are-tabs).
```

Comment:
Can you check if the url works without the en-us? If so use that since it will presumably redirect to localized help if appropriate.